### PR TITLE
Update the dependency column for Usage rules. 

### PR DESCRIPTION
**Pull request by mavasani**

### CODE REVIEW

**Comment by mavasani** [comment]
@srivatsn @lgolding @michaelcfanning @genlu

File: RulesInventory.csv [hunk 1]
```diff
@@ -86,37 +86,37 @@ CA1033,InterfaceMethodsShouldBeCallableByChildTypes,Microsoft.Api.Design.Guideli
...
 CA1031,DoNotCatchGeneralExceptionTypes,Microsoft.Api.Design.Guidelines,#N/A,Do not catch general exception types,"General exceptions should not be caught. Catch a more specific exception, or rethrow the general exception as the last statement in the catch block.",Yes,2802219,35947,0.012828048,502.6097536,Design,Low,,,None (similar CA2153 already ported)
-CA1062,#N/A,Microsoft.Api.Design.Guidelines,#N/A,Validate arguments of public methods,All reference arguments that are passed to externally visible methods should be checked against null.,Yes,2267308,29134,0.0128496,494.6076696,Design,Low,,,Dataflow
-CA2224,OverrideEqualsOnOverloadingOperatorEquals,Microsoft.Api.Design.Guidelines,#N/A,Override Equals on overloading operator equals,A public type implements the equality operator but does not override Object.Equals.,Yes,7619,11,0.001443759,2688.743696,Usage,High,Merge?,,
+CA1062,ValidateArgumentsOfPublicMethods,Microsoft.Api.Design.Guidelines,#N/A,Validate arguments of public methods,All reference arguments that are passed to externally visible methods should be checked against null.,Yes,2267308,29134,0.0128496,494.6076696,Design,Low,,,Dataflow
```
Comment (mavasani) [inline RulesInventory.csv]: Also added "Name" field for few rules with N/A

**Comment by michaelcfanning** [comment]
LGTM
can I get a crack at updating this file next?? :)

**Comment by mavasani** [comment]
@michaelcfanning Yes, its all yours :)

**Pull request merged**